cloudbuild.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+
+# Container to publish git repositories over HTTP.
+- name: 'gcr.io/cloud-builders/docker'
+  args: [
+      'build', '--tag', 'git-http-backend',
+      'https://github.com/v1bri/git-http-backend.git#git-daemon-package' ]
+
+# Docker compose container.
+- name: 'gcr.io/cloud-builders/docker'
+  args: [
+      'build', '--tag', 'docker-compose', '--build-arg', 'version=1.23.2',
+      'https://github.com/GoogleCloudPlatform/cloud-builders-community.git#:docker-compose' ]
```

Comment:
What is this URL supposed to do?

Reply:
The `docker build` command supports git repository URLs [1]. The fragment on the end lets you specify a branch and a subfolder to look for the Dockerfile. This the URL to the Docker Compose container that runs the integration test in the final step.

[1] https://docs.docker.com/engine/reference/commandline/build/#git-repositories